README.md

```diff
@@ -75,24 +65,21 @@ Yup, I do:
 ```
 Framework            Requests/sec  % from best
 ----------------------------------------------
-rack                     18544.85       100.0%
```

Comment:
Please do not update README numbers, these numbers will be updated using the same hardware that was originally published.

Want to keep the hardware used for the benchmark consistent. Other developers have sent their benchmarks but is very specific of their machines.